myanmar-phonenumber.js

```diff
@@ -125,7 +130,7 @@
         phoneNumber = this.normalizeInput(phoneNumber);
         if (operatorRe.ooredoo.test(phoneNumber) || operatorRe.telenor.test(phoneNumber) || operatorRe.mytel.test(phoneNumber)) {
           networkType = this.NETWORK_TYPE.GSM;
-        } else if (operatorRe.mpt.test(phoneNumber)) {
+        } else if (operatorRe.mpt.test(phoneNumber) || operatorRe.mec.test(phoneNumber)) {
```

Comment:
Shed me a light on this. MEC doesn't have its own dedicated network type?

Reply:
MEC only have CDMA 800 type and the regex for it is already in MPT's if case so I just added an OR. We can also do a separate else if case for MEC with only CDMA regex if the former approach is confusing.